client/src/i18n/index.js

```diff
@@ -164,6 +164,141 @@ const factory = (options = { debug: true }) =>
             },
           },
         },
+        pl: {
+          translation: {
+            objectives: objectivesPl,
+            general: {
+              switchLanguage: 'Zmień język',
+              home: 'Home',
+              title: 'TI4 Companion',
+              labels: {
+                add: 'dodaj',
+                cancel: 'anuluj',
+                ok: 'ok',
+                objective: 'cel',
+                search: 'szukaj',
+                stageI: 'poziom I',
+                stageII: 'poziom II',
+                secretObj: 'sekretny',
+                save: 'zapisz',
+                undo: 'cofnij',
+                copy: 'kopiuj',
+              },
+              phase: {
+                0: 'faza statusu',
+                1: 'faza akcji',
+                2: 'faza polityki',
+              },
+              reward: {
+                0: 'punkt zwycięstwa',
+              },
+              confirmation: {
+                title: 'Jesteś pewny?',
+              },
+            },
+            sessionView: {
+              nav: {
+                overview: 'Podsumowanie',
+                map: 'Mapa',
+                details: 'Szczegóły',
+              },
+              overview: {
+                goToWiki: 'przejdź do wiki',
+                openOriginal: 'otwórz oryginalny obraz',
+                sessionStart: 'sesja od {{date}} {{time}}',
+              },
+            },
+            sessionMap: {
+              map: 'mapa TI4',
+              none: 'mapa nie została wszytana',
+              changeFile: 'Zmien plik mapy',
+              dropHere: 'Przeciągnij mapę tutaj...',
+              dragHere: 'Przeciądnij i upuść , or click to select the map file',
+              sizeHint:
+                'Miej na uwadze, że mapy mniejsze niż 800x800px mogą być nieczytelne',
+              preview: 'preview',
+            },
+            share: {
+              copied: 'Skopiowano!',
+              tooltip: 'pokaż kod qr',
+              allowEdit: 'zezwól na edycję',
+            },
+            sessionDetails: {
+              name: 'Nazwa Twojej sesji',
+              tts: 'TTS',
+              split: 'Split',
+              startDate: 'Data rozpoczęcia sesji',
+              endDate: 'Data zakończenia sesji',
+              duration: 'Jak długo trwała sesja? (roughly)',
+              durationUnit: 'godziny',
+              detailsSavedCorrectly: 'Detale zapisane pomyślnie',
+              vpChangeConfirmation: {
+                title: 'Zmieniłeś ilość punktów zwycięstwa',
```

Comment:
```suggestion
                title: 'Ilość punktów zwycięstwa uległa zmianie',
```
let's use non-gender-specific forms